vscode_extension/src/test/suite/aacExecutableWrapper.test.ts

```diff
@@ -13,7 +13,7 @@ suite("AaC Executable Wrapper Test Suite", () => {
         await aacWrapper.executeCommandWithArguments(versionCommand)
             .then((output: string) => {
                 // Leaving the minor version out of the test.
-                const searchString = "0.2.";
+                const searchString = "0.3.";
```

Comment:
This is the test you were referring to in your bug #682. I don't know off hand if/how this would need swapped out. I'll poke around about it tomorrow.

Reply:
This was certainly a surprise that popped up in the CI/CD pipeline and is the reason I wrote that bug.  It was super frustrating to have run the 10 min + tox runs to try and quash all errors only to be hit with this post-commit.